### ISSUE TITLE
Add ACML Acceptance Rates?

### ISSUE THREAD

**Issue by alexandertornede** [lixin4ever/Conference-Acceptance-Rate#30]
Hi, 
would it be possible to add the ACML acceptance rates: http://acml-conf.org/Print.aspx?Page=MainPage

I can also file a pull request with the required data, if requested. 

Best regards!

**Comment by ZaydH** [comment]
I agree with @alexandertornede. ACML is growing in prominence with increasingly good research published there and prominent PCs (e.g., Masashi Sugiyama).

**Comment by henrygouk** [comment]
I'd argue that ECML should be even higher priority than ACML. I've attended both conferences a couple of times and ECML is noticeably bigger---around 1,000 attendees last year.

**Comment by ZaydH** [comment]
@henrygouk Agree.  ECML is more prestigious and established than ACML.

**Comment by alexandertornede** [comment]
I agree with both of you that ECML is more established than the ACML. I am not sure if they also publish their rates in a collection as ACML does or if anyone has that data. If so, I guess both could be added.
